packages/bpk-component-chip/src/BpkSelectableChip.tsx

```diff
@@ -64,7 +65,7 @@ const BpkSelectableChip = ({
       {...rest}
     >
       {leadingAccessoryView && (
-        <span className={getClassName('bpk-chip__leading-accessory-view')}>
+        <span className={getClassName('bpk-chip__leading-accessory-view', leadingAccessoryClassName)}>
```

Comment:
I'm thinking whether we should introduce a new className prop instead of modifying component own styles? Looks like a source of potential non-determinism problems

Reply:
How would you go about doing this? The styles we need to add override the default chip behaviour - this should work for the BpkIconChip afaik but does open up the potential for non-determinism problems if consumers start using this new prop

Reply:
I've removed the leadingAccessoryClassname prop and just optionally add the icon-only if the children are not set - this seemed slightly better to me than having a flag that should only be used by `BpkIconChip`, what do you think?